mission_control/navigator_launch/launch/cameras.launch

```diff
@@ -0,0 +1,80 @@
+<launch>
+  <!-- This is the default pointgrey launch file with our id's and our names  -->
+
+  <arg name="front_camera_serial" default="15223362" />
+  <arg name="front_camera_calibrated" default="0" />
+  <arg name="down_camera_serial" default="10282860" />
+  <arg name="down_camera_calibrated" default="0" />
```

Comment:
Werid... These are PointGrey launches, not dc1394 ones. I'm not entirely sure why these are still around. These are not the launches that are running the cameras, and if this is what's being used to launch we need to switch over to the launches for dc1394.